publish-subscribe/src/main/java/com/iluwatar/publish/subscribe/model/TopicName.java

```diff
@@ -0,0 +1,7 @@
+package com.iluwatar.publish.subscribe.model;
+
+/** This enum defines the available topic names to be used in {@link Topic}. */
+public enum TopicName {
+  WEATHER,
+  CUSTOMER_SUPPORT
+}
```

Comment:
Not sure why we need to limit the topic names to these? Wouldn't it be more flexible to allow any name?